pkg/vm/stackitem/stack_item.go

```diff
@@ -11,11 +11,13 @@ import (
 	"reflect"
 
 	"github.com/nspcc-dev/neo-go/pkg/crypto/hash"
-	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
 	"github.com/nspcc-dev/neo-go/pkg/util"
-	"github.com/nspcc-dev/neo-go/pkg/vm/emit"
+	"github.com/nspcc-dev/neo-go/pkg/vm/bigint"
 )
 
+// MaxBigIntegerSizeBits is the maximum size of BigInt item in bits.
+const MaxBigIntegerSizeBits = 32 * 8
+
 // A StackItem represents the "real" value that is pushed on the stack.
 type StackItem interface {
```

Comment:
I think we can make an `Item` now, `stackitem.Item` is way better than `stackitem.StackItem`.